.clang-format

```diff
@@ -7,10 +7,13 @@ Language:        Cpp
 #   - IndentWidth: 2 -> 4
 #   - SpacesInParentheses: false -> true
 #   - BreakConstructorInitializersBeforeComma: false -> true
+#   - IndentPPDirectives: None -> AfterHash
+#   - IncludeBlocks: Preserve -> Regroup
 AccessModifierOffset: -2
-AlignAfterOpenBracket: true
```

Comment:
@Rombur I generated new LLVM style config file and applied the changes you reported.
It looks like `AlignAfterOpenBracket` and `AlignEscapedNewlinesLeft` differ.  Do you know if it is something you did and forgot to write down or an actual change in the LLVM style?

Reply:
They change their style. I made sure at the time to record the changes I made